resources/production/config-example.toml

```diff
@@ -44,7 +44,10 @@ unit_hashes_folder = "/var/lib/casper/casper-node"
 pending_vertex_timeout = '30min'
 
 # If the current era's protocol state has not progressed for this long, shut down.
-# standstill_timeout = '30min'
+standstill_timeout = '30min'
+
+# If after another `standstill_timeout` there still was no progress, shut down.
+shutdown_on_standstill = true
```

Comment:
Sorry, this option was removed actually removed in https://github.com/casper-network/casper-node/pull/1989: The shutdown is now enabled simply by specifying a `standstill_timeout`.